docs/src/pages/components/slider/slider.md

```diff
@@ -77,6 +77,16 @@ Here are some examples of customizing the component. You can learn more about th
 
 {{"demo": "pages/components/slider/VerticalSlider.js"}}
 
+**WARNING**: Chrome, Safari and newer Edge versions i.e. any browser based on WebKit exposes `<Slider orientation="vertical" />` as horizontal ([chromium issue #1158217](https://bugs.chromium.org/p/chromium/issues/detail?id=1158217)).
+By applying `-webkit-appearance: slider-vertical;` the slider is exposed as vertical.
+
+However, by applying `-webkit-appearance: slider-vertical;` keyboard navigation for horizontal keys (`ArrowLeft`, `ArrowRight`) is reversed ([chromium issue #1162640](https://bugs.chromium.org/p/chromium/issues/detail?id=1162640)).
```

Comment:
```suggestion
However, by applying `-webkit-appearance: slider-vertical;` keyboard navigation for horizontal keys (<kbd>ArrowLeft</kbd>, <kbd>ArrowRight</kbd>) is reversed ([chromium issue #1162640](https://bugs.chromium.org/p/chromium/issues/detail?id=1162640)).
```

Reply:
With `kbd` we want the actual keys. `ArrowLeft` refers to the value of `event.key`.

```suggestion
However, by applying `-webkit-appearance: slider-vertical;` keyboard navigation for horizontal keys (<kb>←</kbd>, <kb>→</kbd>) is reversed ([chromium issue #1162640](https://bugs.chromium.org/p/chromium/issues/detail?id=1162640)).
```

Reply:
Would it work with a space to match what WAI ARIA is doing? I'm not sure that `←` is obvious enough

```suggestion
However, by applying `-webkit-appearance: slider-vertical;` keyboard navigation for horizontal keys (<kbd>Arrow Left</kbd>, <kbd>Arrow Right</kbd>) is reversed ([chromium issue #1162640](https://bugs.chromium.org/p/chromium/issues/detail?id=1162640)).
```

Also note this precedence: https://material-ui.com/components/data-grid/accessibility/#navigation

Reply:
> Would it work with a space to match what WAI ARIA is doing? 

That's an actual argument? Why do we also need a review roundtrip for that?

Reply:
Resolved with [`552f608` (#23902)](https://github.com/mui-org/material-ui/pull/23902/commits/552f60851886d00529fffa4eb996ce3e30d9f2f2)

Reply:
> Why do we also need a review roundtrip for that?

Because I didn't think about it in the first place :)